contracts/PublicResolver.sol

```diff
@@ -52,4 +53,14 @@ contract PublicResolver is ABIResolver, AddrResolver, ContentHashResolver, DNSRe
         address owner = ens.owner(node);
         return owner == msg.sender || authorisations[node][owner][msg.sender];
     }
+
+    function multicall(bytes[] calldata datas) external returns(bytes[] memory rets) {
+        rets = new bytes[](datas.length);
+        for(uint i = 0; i < datas.length; i++) {
+            (bool success, bytes memory ret) = address(this).delegatecall(datas[i]);
+            require(success);
```

Comment:
What is our policy of adding error message as a second parameter? It's inconsistent so far.

Reply:
I've been omitting it, because error messages cause significant bloat.